src/lib.rs

```diff
@@ -435,6 +435,7 @@ impl Capture<Offline> {
 
     /// Opens an offline capture handle from a pcap dump file, given a path.
     /// Takes an additional precision argument specifying the time stamp precision desired.
+    #[cfg(pcap_1_5_0)]
```

Comment:
(Just a name nitpick)
We are preparing the crate `pcap-0.8.0` now, but will eventually see greater version numbers of our crate.
`pcap_1_5_0` might be confusing when we'll release `pcap-1.5.0` crate in future.

Suggestion: Use `libpcap` name when we refer to the C library we are wrapping.
* `pcap_1_5_0` -> `libpcap_1_5_0`
* `pcap_1_7_2` -> `libpcap_1_7_2`
* etc.

Reply:
Good idea, done